### PR TITLE
Add access token login

### PR DESCRIPTION
**Pull request by gcandal**

### CODE REVIEW

**Comment by gcandal** [comment]
@joamag

File: src/ripe/base.py [hunk 1]
```diff
@@ -63,18 +64,29 @@ def auth_callback(self, params, headers):
...
-        url = self.base_url + ("signin_admin" if admin else "signin")
-        contents = self.post(
-            url,
-            callback = False,
-            auth = False,
-            username = username,
-            password = password
-        )
+    def login(self, access_token = None, username = None, password = None, admin = None):
+        access_token = access_token or self.access_token
```
Comment (joamag) [inline src/ripe/base.py:68]: the name is just token and not access_token

**Comment by joamag** [comment]
@gcandal check theses changes https://github.com/ripe-tech/ripe-api/commit/804238ebf77d40b5390428d1deba7c91a0bd3e35 I've split the code to simplify it

**Comment by gcandal** [comment]
> @gcandal check theses changes [804238e](https://github.com/ripe-tech/ripe-api/commit/804238ebf77d40b5390428d1deba7c91a0bd3e35) I've split the code to simplify it

@joamag shouldn't `get_session_id` be altered too to fallback to `login_sid` if there's a `token`?

**Comment by joamag** [comment]
> > @gcandal check theses changes [804238e](https://github.com/ripe-tech/ripe-api/commit/804238ebf77d40b5390428d1deba7c91a0bd3e35) I've split the code to simplify it
> 
> @joamag shouldn't `get_session_id` be altered too to fallback to `login_sid` if there's a `token`?

You're right just fixed that https://github.com/ripe-tech/ripe-api/commit/b3b10767d4da8593ec7fd8d0255f66de577d8301

**Pull request merged**